src/merge_sort.py

```diff
@@ -0,0 +1,78 @@
+"""Merge Sort Module."""
+
+# MERGE SORT (MS)
+#
+# CodeFellows 401d5
+# Submission Date:
+#
+# Authors:  Colin Lamont <https://github.com/chamberi>
+#           Ben Shields <https://github.com/iamrobinhood12345>
+#
+# URL:
+
+
+def merge_sort(msl):
+    """Merge sort method."""
+    if len(msl) == 1 or not msl:
+        return msl
+    msl1 = msl[:int(len(msl) / 2)]
+    msl2 = msl[int(len(msl) / 2):]
+
+    msl1 = merge_sort(msl1)
+    msl2 = merge_sort(msl2)
+
+    def _merge(msla, mslb):
+        """Merge compares the two lists and returns a sorted list from lowest to highest value.""" 
+        sorted_list = []
+        while len(msla) and len(mslb):
+            if msla[0] < mslb[0]:
+                low = msla.pop(0)
+            else:
+                low = mslb.pop(0)
+            sorted_list.append(low)
+        if len(msla):
+            sorted_list.extend(msla)
+            return sorted_list
+        else:
+            sorted_list.extend(mslb)
+            return sorted_list
+
+    return _merge(msl1, msl2)
+
+
+def _random_list():
```

Comment:
You could one line this.  If you are going to bother to make this a function, why not make it so you can pass in an integer n, and gen a list of length n?